src/es-features/task5.js

```diff
@@ -16,4 +16,6 @@ export function task5Old(array, obj) {
 // Напишите реализацию функции task5Old на ES6+ ниже этого комментария.
 // При желании, можете использовать стрелочную функцию, вместо обычной
 
-export function task5New() {}
+export const task5New = (array, obj) => [...array, ...Object.keys(obj)];
```

Comment:
your code changed logic that was written in an old variant. Your code shouldn't change the behavior of the function.